entity-api-spec.yaml

```diff
@@ -1991,23 +2284,19 @@ paths:
           required: false
           schema:
             type: string
-            enum: [ 'true', 'false' ]
+            enum: ['true', 'false']
       responses:
         '200':
           description: The list of revised datasets that the referenced dataset is a member of including the index number of the revision, where 1 is the oldest version of any revision chain
           content:
             application/json:
               schema:
                 type: object
-                properties:
-                  uuid:
-                    type: string
-                    description: The uuid of a dataset
-                  revision_number:
-                    type: integer
-                    description: The number in the revision chain of this dataset where 1 is the oldest revision
-                  dataset:
-                    $ref: '#/components/schemas/Dataset'
+              examples:
+                revisionexamples:
+                  value: 
+                    - 'abcd1234-ef56-gh78-ij90-klmnop123456' 
+                    - 'abcd1234-ef56-gh78-ij90-klmnop123456'        
```

Comment:
The result of `GET /datasets/{id}/revisions` is a json array of objects, and each object has `uuid` and `revision_number`, not just a string.